data/keys.go

```diff
@@ -5,7 +5,7 @@ import (
 	"encoding/hex"
 
 	"github.com/Sirupsen/logrus"
-	cjson "github.com/tent/canonical-json-go"
+	"github.com/jfrazelle/go/canonical/json"
```

Comment:
Ignore the above; I just noticed that the marshal function is called `MarshalCanonical`.